src/desktop/components/main_layout/public/icons/wechat.svg

```diff
@@ -1,50 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 316 254.9" style="enable-background:new 0 0 316 254.9;" xml:space="preserve">
 <style type="text/css">
-	.st0{filter:url(#Adobe_OpacityMaskFilter);}
-	.st1{fill:#FFFFFF;}
-	.st2{mask:url(#mask-2_2_);}
-	.st3{filter:url(#Adobe_OpacityMaskFilter_1_);}
-	.st4{mask:url(#mask-4_2_);}
+	.wechat__primary-filter{filter:url(#WeChat_OpacityMaskFilter);}
```

Comment:
I left a few of the selectors as `#id` because they were being used as urls like above.